### PR TITLE
Bayonetta Hotfix: Jump, Effects, Misc.

### PR DESCRIPTION
**Pull request by jaccob101**
Fixes several bugs and inconsistencies, many due to user error on github desktop. Includes code review by JOB. Some of the fixes include adopting modern hdr standards and fixing infinite jump glitches.

Side B: 
~ [R] Angle setting uses floats, remains using wind energy for time being (to be adjusted on a later date?)
~ [R] DABK Attack macro removed
~ [R] Shortening based on angle removed until/unless helped making it work again.
~ [R] Now able to slide off while hitbox is out as long as it does not hit a shield, rather than unclear frame window
~ [+] ABK BKB: 60/55 -> 64/58 (first hits)
~ [+] ABK KBG: 30 -> 27 (first three hits) 

Up B: 
Angle and set_pos were mistakenly not compensated for aerial ver having shorter distance and hitting grounded opponents
~ [\] Angle: 367 -> 366 on multihit (no longer launches grounded opponents at 80% angle)
~ [\] Set_pos Y offset: 35.0 -> 35.5 grounded, 34.0 aerial
~ [\] Wind energy only applied if finisher will hit, no longer does strange float on whiff
~ [+] BKB: 55 -> 50 (grounded, less punishing)

Up Tilt: 
Accidentally reverted changes corrected and readjusted. 
~ [R] Angles: Inner hit 123 -> 110 (2 degrees steeper than Askew's last fix), 128-> 108 inner and 138/150 outer
~ [R] FKB: 145 -> 100 inner hit (matches other inner hit), 100 -> 125 outer hit (formerly intended change but weakened)
~ [R] BKB: 65 -> 60 inner hit and outer grounded hitbox, 65 -> 75 air only outer hit
~ [R] Inner hitbox now able to reverse hit, people hit from behind her will now get hit in front of her instead of flinging behind her.
~ [R] New air only hitbox added, fixes fallouts and prevents people on platforms from sliding like crazy

Forward Tilt: 
Rebalanced to make both options more even in terms of risk and combo-ability. Each option has its trade-offs
Ftilt 1:
~ [R] Ftilt 2 Combo frame: 11 -> 10
~ [R] Ftilt 3 Combo frame 9 -> 10 (standardization / fixes)

Ftilt 2:
~ [+] Beta Hitboxes restored, blindspot during leg swing back fixed (z-axis?? plus it not reaching heel)
~ [/] SFX now match Ftilt 3
~ [/] Angle: 66 -> 62/67 (nightly inner hit sends at a way higher angle, as close to even as i could get eyeballing)
~ [+] BKB: 81 -> 77 (compensate for no longer having that weird high angled inner hit)
~ [R-] Startup: 10 -> 12
~ [R-] FAF: 26 -> 29 
~ [R-] KBG: 50 -> 52 

Ftilt 3:
~ [R] C-stick up inputs now register as ftilt 3 inputs instead of needing to input with left stick 
~ [R+] Startup: 15 -> 12
~ [R+] FAF: 42 -> 39
~ [R-] KBG: 60 -> 63 

Jab: 
~ [/] Inputs fixed to match similar jabs such as Corrin's. Hold works if it hits anything and now does jab 1->3 combo, release and mash A for rapid jab (standard). Also now able to rapid jab without hitting someone
Jab 3:
~ [/] Angle: 55 -> 50/55 (nightly inner hits send at way higher angle, roughly even now)

Heel Slide Kick:
~ [+] BKB: 64 -> 60 (revert of prev nerf, faf nerf was enough)

### CODE REVIEW

**Comment by jaccob101** [comment]
I believe i've fixed all the bugs i've noticed and knew how to fix, lmk if I made any errors or if theres a better way to fix things

File: fighters/bayonetta/src/acmd/tilts.rs [hunk 1]
```diff
@@ -113,15 +113,16 @@ unsafe fn bayonetta_attack_hi3_game(fighter: &mut L2CAgentBase) {
...
+        ATTACK(fighter, 2, 0, Hash40::new("armr"), 2.0, 135, 27, 0, 75, 5.0, 6.0, 0.0, 0.0, None, None, None, 0.8, 0.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_F, true, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_A, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_PUNCH);
+        ATTACK(fighter, 3, 0, Hash40::new("armr"), 2.0, 135, 27, 0, 60, 5.0, 6.0, 0.0, 0.0, None, None, None, 0.8, 0.0, *ATTACK_SETOFF_KIND_ON, *ATTACK_LR_CHECK_F, true, 0, 0.0, 0, false, false, false, false, true, *COLLISION_SITUATION_MASK_GA, *COLLISION_CATEGORY_MASK_ALL, *COLLISION_PART_MASK_ALL, false, Hash40::new("collision_attr_normal"), *ATTACK_SOUND_LEVEL_S, *COLLISION_SOUND_ATTR_PUNCH, *ATTACK_REGION_PUNCH);
```
Comment (SkewedAskew) [inline fighters/bayonetta/src/acmd/tilts.rs]: While id2 does override this hitbox when in contact with aerial opponents, please change id3 to ground-only for consistency

**Review by SkewedAskew — CHANGES_REQUESTED** [review]

**Comment by jaccob101** [comment]
Ok got everything I can do on my own done. Includes the code review. I can't figure out how to shorten ABK or make ftilt 3 work if I check for special input. Also struggled trying to make the effects clear on the results screen

**Review by SkewedAskew — CHANGES_REQUESTED** [review]
- Performing a shorthop, inputting ABK on the first frame of being airborne, and then pressing down on frame 2 of the move will still perform DABK
- Ftilt3 is actually 36 FAF, not 37
- Ftilt3 can no longer be performed using Special

**Comment by jaccob101** [comment]
> * Performing a shorthop, inputting ABK on the first frame of being airborne, and then pressing down on frame 2 of the move will still perform DABK
> 
>     * Ftilt3 is actually 36 FAF, not 37
> 
>     * Ftilt3 can no longer be performed using Special

i cant find out how to fix the first one but i fixed the ftilt 3 special input by changing motion and disabling the combo flag (kept forcing ftilt 2)

UPDATE: changes make the first point no longer unintended

**Review by hdr-balance-backroom — DISMISSED** [review]
- Revert ftilt 3's endlag decrease (the reduced startup is fine, FAF should be 39 as a result)
- Remove the ability to hold special to not slide off ledge during Heel Slide

**Comment by jaccob101** [comment]
> - Revert ftilt 3's endlag decrease (the reduced startup is fine, FAF should be 39 as a result)
> - Remove the ability to hold special to not slide off ledge during Heel Slide

Done, my bad. I just realized that the heelslide kick status is prob separate anyways so it'll prob make more sense that way anyways

**Review by SkewedAskew — APPROVED** [review]

**Pull request merged**